stix_shifter_modules/crowdstrike_logscale/stix_translation/json/crowdstrike_edr_from_stix_map.json

```diff
@@ -0,0 +1,336 @@
+{
```

Comment:
don't think you need `crowdstrike` in the dialect name. just edr should be enough.  crowdstrike_edr_from_stix_map.json -> edr_from_stix_map.json

same goes to to_stix mapping file

Reply:
Based on the discussion, since Crowdstrike EDR doesn't have common schema, product name is maintained as part of dialect(mapping file names) name. Similarly, the dialect name is updated without underscore(crowdstrikeedr) in mapping files.